src/services/completions.ts

```diff
@@ -598,8 +619,55 @@ namespace ts.Completions {
             has: name => uniques.has(name),
             add: name => uniques.set(name, true),
         };
+
+        function shouldIncludeSymbol(symbol: Symbol, symbolToSortTextMap: SymbolSortTextMap): boolean {

```

Comment:
This was just moved from a separate filter statement elsewhere into this spot where we already iterate through the whole list. Should be a small performance improvement, but I moved it because of some order-of-operations change that I honestly don’t remember anymore.